week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,18 @@
+function percentage(numStudents, total) {
+  var numStudents = 15;
+
+  var total = 23;
+  var percentageofStudent = Math.round((numStudents / total) * 100);
+
+  return percentageofStudent + "%";
+}
+function Percentage(numMentors, total) {
```

Comment:
Nice one. Consider though, that naming matters, you are using the same name for two functions with a case difference. Of course, you remember now what each one of this, is returning, but think about naming them appropriately, so that the names represent in a better way the responsibility of its function, in order to avoid any problems in the future, by using the wrong one.